api/proto/info/info.proto

```diff
@@ -507,6 +509,40 @@ message ZInfoNetworkInstance {
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastRefcountChangeTime = 3; // Last time when refCount changed
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
+
+message ZInfoVolume {
+  string uuid = 1;
+  string displayName = 2;
+  UsageInfo usage = 3;
+  VolumeResources resources = 4;
+  ZSwState state = 5;
+  uint32 downloadProgress = 6;
+
+  ErrorInfo volumeErr = 7;
+}
+
+message ContentResources {
+  uint64 curSizeBytes = 1;  // Current disk usage
+}
+
+message ZInfoContentTree {
+  string uuid = 1;
```

Comment:
Same question here about who controls the lifecycle of these UUIDs @eriknordmark @bharani-zededa

Reply:
The controller. We currently have ImageIDs from the controller for this.
Hmm - and an imageID which refers to e.g., nginx:latest might refer to different contentTrees if it was resolved at different times for different app instances/volumes resulting in different hashes.